helm/hello-world/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.serviceMonitor.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    application.giantswarm.io/team: {{ .Values.serviceMonitor.team }}
+    app.kubernetes.io/instance: {{ .Values.serviceMonitor.instance }}
+  name: {{ include "hello-world.fullname" . }}
+spec:
+  endpoints:
+  - interval: 60s
+    path: /metrics
+    port: http
+  selector:
+    matchLabels:
+      {{- include "hello-world.labels" . | nindent 8 }}
+{{- end }}
```

Comment:
It renders something like
```apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  labels:
    application.giantswarm.io/team: team-awesome
    app.kubernetes.io/instance: hello-world
  name: fer-hello-world
spec:
  endpoints:
  - interval: 60s
    path: /metrics
    port: http
  selector:
    matchLabels:
        helm.sh/chart: hello-world-2.3.2
        app.kubernetes.io/name: hello-world
        app.kubernetes.io/instance: fer
        app.kubernetes.io/version: "0.2.1"
        app.kubernetes.io/managed-by: Helm
        giantswarm.io/service-type: managed
        application.giantswarm.io/team: "honeybadger"
```
But still don't see anything in `Explore` tab on grafana. Any idea why?

Reply:
Can you try to check on the prometheus agent ui by port-forwarding? Where is it déployed?

Reply:
Now it works perfect